tutorials-dp0-3/portal-dp0-3-3.rst

```diff
@@ -0,0 +1,425 @@
+
+.. code-block:: SQL 
+
+    SELECT ds.mag, ds.band, ds.midPointMjdTai, 
+    ss.phaseAngle, ss.topocentricDist, ss.heliocentricDist 
+    FROM dp03_catalogs_10yr.DiaSource AS ds 
+    JOIN dp03_catalogs_10yr.SSSource AS ss ON ds.diaSourceId = ss.diaSourceId
+    WHERE ss.ssObjectId = -735085100561880491
+    AND ds.band = 'r'
+
+3.2. Use the plot "Settings" function to add new scatter plots showing the r-band magnitude and phase angle
+as a function of time (right two plots, below), and see that these quantities are not correlated with time.
+Add a new scatter plot showing the r-band magnitude as a function of phase angle, which are correlated.
+
+.. figure:: /_static/portal_tut03_step03a.png
+    :name: portal_tut03_step03a
+    :width: 600
+    :alt: A screenshot of three plots showing magnitude and phase angle are not correlated with time, and that magnitude is correlated with phase angle.
+
+    Three plots demonstrating that magnitude and phase angle are correlated with each other, but not with time.
+
+3.3.  Plot the topocentric and heliocentric distances of the object as a function of time.  
+Here, you will use the columns in the table which you generated in Step 3.1.  
+First, delete two of the the three plots prepared in Step 3.2 by clicking on the blue ``X`` in the upper right-hand part of the plot panels to make space for new plots.  
+Then add a pair of new plots, clicking on the "plot settings."  
+In both cases, enter ``midPointMjdTai`` for X-axis, but for Y axis - enter ``topocentricDist`` for one plot, and ``heliocentricDist`` for the other.  
+After you remove the panel containing the plot made in the previous step, you will see the plots as below.  
+
+.. figure:: /_static/portal_tut03_step03b.png
+    :width: 600
+    :name: portal_tut03_step03b
+    :alt: A screenshot of two plots showing the heliocentric and topocentric distance of the trans-Neptunian object as a function of time.
+
+    Note the periodic change of the topocentric distance with time resulting from the Earth's motion around the Sun - a different view of the same effect you saw in Step 2.5.  
+
+.. _DP0-3-Portal-3-Step-4:
+
+Step 4. Visualize the TNO's trajectory in 3-D 
+=============================================
+
+4.1.  Execute the query below to extract the helio- and topocentric distances of the TNO - so you can visualize its trajectory:  
+
+.. code-block:: SQL 
+
+    SELECT heliocentricX, heliocentricY, heliocentricZ,
+    topocentricX, topocentricY, topocentricZ, ssObjectId
+    FROM dp03_catalogs_10yr.SSSource
+    WHERE ssObjectId = -735085100561880491
+
+4.2.  Plot the heliocentric Z distance as a function of heliocentic X distance by clicking on the "plot setings" icon and selecting ``heliocenticZ`` for y and ``heliocentricY`` for x.  
+Note that the object moves relatively slowly in heliocentric coordinate X (as well as in Y), covering only a few au in 10 years.  
```

Comment:
The "relatively" here will be hard for the user to understand. Relative to what? There is only one object being plotted. Reword.

Reply:
OK